### PR TITLE
added protocol 407 for MCPE 1.16

### PR DESCRIPTION
**Pull request by suerion**

### CODE REVIEW

**Comment by jasonw4331** [comment]
Tested?

**Comment by suerion** [comment]
> 
> 
> Tested?

Yes, no problems, with windows 10,

xbox looks a little bit strange, but it work

File: plugin.yml [hunk 1]
```diff
@@ -5,5 +5,7 @@ api: 3.0.0
 mcpe-protocol:
  - 389
  - 390
+ #Added Protocol 407 for MCPE 1.16
+ - 407
 description: "A plugin which adds commands to the client's autofill by parsing usage messages"
-author: "jasonwynn10"
\ No newline at end of file
+author: "jasonwynn10"
```
Comment (jasonw4331) [inline plugin.yml]: new line at end of file needs removed

File: plugin.yml [hunk 2]
```diff
@@ -5,5 +5,7 @@ api: 3.0.0
 mcpe-protocol:
  - 389
  - 390
+ #Added Protocol 407 for MCPE 1.16
```
Comment (jasonw4331) [inline plugin.yml]: please remove the comment for the protocol

**Review by jasonw4331 — CHANGES_REQUESTED** [review]

**Comment by suerion** [comment]
should be resolved

**Comment by jasonw4331** [comment]
There's still a new line at the end of the file

**Comment by suerion** [comment]
i deleted it allways, but it add it after commit the pull request, weired

**Comment by HimmelKreis4865** [comment]
I had actually the same problem as you, maybe ur using CRLF instead of LF line separator.

**Comment by jasonw4331** [comment]
seriously dude, fix your line endings...

**Comment by suerion** [comment]
i will close this pull request, i hope you could say me, whats the problem, i work with eclips and the lf line seperator, but it does not chenge anything...

**Pull request closed without merging**